BibLaTeX.js

```diff
@@ -386,7 +386,12 @@ function creatorCheck(item, ctype) {
 	return false;
 }
 
-	function buildCiteKey(item, citekeys) {
+function buildCiteKey (item, extraFields, citekeys) {
+		const citationKey = extraFields.findIndex(field => field.field && field.value && field.field.toLowerCase() === 'citation key')
```

Comment:
why is this `citation key` and not `citationkey` ? (I might be missing something obvious.)

Reply:
As far as I can tell, https://github.com/zotero/translators/blob/master/BibTeX.js#L146 or anywhere after doesn't normalize the key, and according to https://github.com/zotero/translators/pull/1810#issuecomment-456219750 the `extra` field would be `Citation Key`. Right? I did test and it seemed to work this way. I can add code to normalize the parsed field keys, but I don't know what else that would break.

Reply:
we don't have a convention on this -- currently, all other fields we use Extra for, we just pick up the CSL variable name (for obvious reasons), but that doesn't exist here.

I don't have a strong view on how this should be named, but to keep it consistent with other fields in extra, should we name it what a CSL variable _would_ be named, i.e. `citation-key`?
How do you currently suggest/require people enter keys in Extra for BBT?

Reply:
I currently support `bibtex:` (for backwards compatibility) and `Citation Key:` (suggested by convention, because it's what BBT will add currently if you pin a key), but will follow any convention that Zotero proposes. If we settle for `citation-key` here, I'll just adjust BBT to pin keys that way. A re-pin would change `bibtex:` or `Citation Key:` to `citation-key` if present under that regime.

Reply:
No need to multiply these then. Let's go with `Citation Key:` i.e. keep the code as is.

Reply:
It's an easy change for me though, so if `citation-key` is a better philosophical fit, that's a 5-minute change for me with no further consequences.

Reply:
`Citation Key:` is good.